test/e2e/baseline_test.go

```diff
@@ -0,0 +1,155 @@
+//go:build integ
+// +build integ
+
+package kmesh
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/echo/common/scheme"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/echo"
+	"istio.io/istio/pkg/test/framework/components/echo/check"
+)
+
+var (
+	callOptions = []echo.CallOptions{
+		{
+			Port:   echo.Port{Name: "http"},
+			Scheme: scheme.HTTP,
+			Count:  10,
+		},
+		{
+			Port:   echo.Port{Name: "tcp"},
+			Scheme: scheme.TCP,
+			Count:  1,
+		},
+	}
+)
+
+func TestKmesh(t *testing.T) {
+	framework.NewTest(t).Run(func(t framework.TestContext) {
+		time.Sleep(100 * time.Second)
+	})
+}
+
+func TestTrafficSplit(t *testing.T) {
+	runTest(t, func(t framework.TestContext, src echo.Instance, dst echo.Instance, opt echo.CallOptions) {
+		// Need at least one waypoint proxy and HTTP
+		if opt.Scheme != scheme.HTTP {
+			return
+		}
+		if !dst.Config().HasServiceAddressedWaypointProxy() {
+			return
+		}
+		if src.Config().IsUncaptured() {
+			// TODO: fix this and remove this skip
+			t.Skip("https://github.com/istio/istio/issues/43238")
```

Comment:
?

Reply:
The test case is direclty copied from ambient integration test (https://github.com/istio/istio/blob/master/tests/integration/ambient/baseline_test.go#L628`).

Actually I think most the test cases there should be copied and used direcly. Because we need to fully match ambient's capabilities and be transparent to isito and users.